tasks/service_scale.json

```diff
@@ -11,7 +11,7 @@
       "description": "Number of replicas",
       "type": "Integer"
     },
-    "detatch": {
+    "detach": {
```

Comment:
ouch :sweat:  since the task already expects the correct spelling, this is a bugfix, not breaking compatibility